src/prefs.h

```diff
@@ -48,8 +48,9 @@ GeanyPrefs;
 /** Tools preferences */
 typedef struct GeanyToolPrefs
 {
-	gchar			*browser_cmd;			/**< web browser command */
 	gchar			*term_cmd;				/**< terminal emulator command */
+	gchar			*file_manager_cmd;		/**< file manager command */
```

Comment:
This struct is in the plugin API, so add new fields at the end to avoid breaking the ABI, increment the API, see HACKING.

Reply:
My bad, will fix